### PR TITLE
[Ready for review] Schema tweaks and sitewide breadcrumbs

### PR DESCRIPTION
**Pull request by frederickobrien**
In hindsight it was really dumb to make one branch for all the stuff I wanted to do. That's what projects are for. I've separated some fairly routine changes into their own branch so they're ready to merge. 

- Site description adjusted
- Array bug fix in nux.config.js
- Featured image schema for articles
- Changed 'Reviews' to 'Album Reviews' on homepage
- Breadcrumbs added to review schema

Nothing major but figured it'd be nice to role them out while Search Console is updating its index of the site. Breadcrumbs were something WP took care of without me realising so I've added them to reviews. Articles will be tricker given the range of content types so I've left that alone for now.

Rich Results Test seems happy enough with the changes:

image

### CODE REVIEW

**Comment by andrewbridge** [comment]
@frederickobrien I've just pushed up an enhancement that generates breadcrumbs for all pages

**Comment by frederickobrien** [comment]
> 
> 
> @frederickobrien I've just pushed up an enhancement that generates breadcrumbs for all pages

All the better! Nice one, @andrewbridge . I think there'll be fine-tuning to be done on speakable markup over time -- it's supposedly for bite-sized content so may not work great for non-review content -- but I'm not sure how best to test it at this stage other than talking to our home speakers and phone assistants.

I [submitted a few URLs to Google for review](https://services.google.com/fb/forms/speakablenewsinterest/) so let's see what they make of it.

Rich Results Test is happy, reckon we're good to go?

**Comment by frederickobrien** [comment]
Oh blimey, I tried adding a 'reviewBody' value to reviews using 'this.review.content' and got a whole lot more structured data nested inside it. Pretty cool to see the individual summaries broken down into their own metadata, though the summaries are still wrapped in HTML tags. 

image

Possibly worth withholding from the merge and devoting a bit more time to, @andrewbridge? Does this have any overlap with your Apple News plans?

**Comment by andrewbridge** [comment]
> Oh blimey, I tried adding a 'reviewBody' value to reviews using 'this.review.content' and got a whole lot more structured data nested inside it.

Yup! This is all from the API, it's all split into sections!

> Possibly worth withholding from the merge and devoting a bit more time to, @andrewbridge? Does this have any overlap with your Apple News plans?

I think between this, and my concerns about size when we stick reviewBody in, it'd be best to leave out for the time being. I know it's not the preferred way, but can we use microdata instead to just markup the content in the HTML?

Re: Apple News (#87), it's kind of related, but only in the sense that we'd take those individually marked up sections and generated the appropriate Apple News markup for it.

File: pages/reviews/_slug.vue [hunk 1]
```diff
@@ -162,6 +162,7 @@ export default Vue.extend({
                     '@type': 'Review',
                     headline: title,
                     description: metadata.summary || metadata.blurb || '',
+                    reviewBody: this.review.content,
```
[the remark this answers is not in the document]
Comment (frederickobrien) [reply pages/reviews/_slug.vue:165]: We _can_ but it's not ideal. Having some bits as JSON and others as microdata/RDFa makes connections unclear. Something to sit on and come back to I guess.

**Comment by frederickobrien** [comment]
> 
> 
> > Oh blimey, I tried adding a 'reviewBody' value to reviews using 'this.review.content' and got a whole lot more structured data nested inside it.
> 
> Yup! This is all from the API, it's all split into sections!
> 
> > Possibly worth withholding from the merge and devoting a bit more time to, @andrewbridge? Does this have any overlap with your Apple News plans?
> 
> I think between this, and my concerns about size when we stick reviewBody in, it'd be best to leave out for the time being. I know it's not the preferred way, but can we use microdata instead to just markup the content in the HTML?
> 
> Re: Apple News (#87), it's kind of related, but only in the sense that we'd take those individually marked up sections and generated the appropriate Apple News markup for it.

Fair enough, I've removed it for the time being. Could the CSS selector approach be a workaround for main content? Says where to look without writing everything out twice.

**Comment by andrewbridge** [comment]
> Fair enough, I've removed it for the time being. Could the CSS selector approach be a workaround for main content? Says where to look without writing everything out twice.

Not sure, the schema seems to be pretty clear about `reviewBody` being

> reviewBody: The actual body of the review.

https://schema.org/Review

You are able to use microdata on the HTML to specify the same attributes. But I know structured data JSON is the preferred metadata method, and I'm unsure how structured data consumers like Google would handle mixed methods.

**Comment by frederickobrien** [comment]
> 
> 
> > Fair enough, I've removed it for the time being. Could the CSS selector approach be a workaround for main content? Says where to look without writing everything out twice.
> 
> Not sure, the schema seems to be pretty clear about `reviewBody`:
> 
> > reviewBody: The actual body of the review.
> 
> https://schema.org/Review
> 
> You are able to use microdata on the HTML to specify the same attributes. But I know structured data JSON is the preferred metadata method, and I'm unsure how structured data consumers like Google would handle mixed methods.

Ok well let's leave it be for now, the site will survive without reviewBody.

**Comment by andrewbridge** [comment]
In that case, do we think this one is ready to go out?

**Comment by frederickobrien** [comment]
> 
> 
> In that case, do we think this one is ready to go out?

I'm happy with it if you are! Deploy seems fine and rich results look good. Would be nice to get the breadcrumbs back in while Google is reindexing the site.

**Review by andrewbridge — APPROVED** [review]
LGTM

**Pull request merged**